code/game/machinery/ecmo.dm

```diff
@@ -0,0 +1,594 @@
+/obj/machinery/ecmo
+	name = "\improper ECMO"
+	desc = "An Extra-Corporeal Membrane Oxygenator."
+	desc_info = "ECMO must be supplied with a blood pack and an oxygen tank."
```

Comment:
I'm going to tentatively disagree with the feedback on changing the name, but I will implore you to write a better description.